spec/requests/posts_spec.rb

```diff
@@ -0,0 +1,23 @@
+require 'rails_helper'
+
+RSpec.describe 'Posts', type: :request do
+  describe 'GET /posts' do
+    before(:example) { get users_path }
+
+    it 'is a success' do
+      expect(response).to have_http_status(:ok)
+    end
+
+    it 'renders the index template' do
+      expect(response).to render_template('index')
+    end
+  end
+
```

Comment:
* According to the project requirement 

> Make sure that for each action you check:

> If the response status was correct.
> If a correct template was rendered.
> If the response body includes correct placeholder text.
* Kindly ensure you write tests to check for all the above in all your actions as some are missing